### PR TITLE
Eclipse contribution microsite

### PR DESCRIPTION
**Pull request by myfear**
The Red Hat Contribution Page. Needs to be reviewed by Max first.

### CODE REVIEW

**Comment by myfear** [comment]
@paulrobinson is there a more elegant way of positioning the hero image? Thanks!

**Comment by paulrobinson** [comment]
Can you be more specific about what the problems are with the current positioning?

**Comment by myfear** [comment]
@paulrobinson atm the following looks ugly:

= define 'hero'
  .row
    .large-12.column
      h2 &amp;nbsp;
    .large-10.column
      p &amp;nbsp;
      p &amp;nbsp;
      p &amp;nbsp;
      p &amp;nbsp;
      p &amp;nbsp;
      p &amp;nbsp;

I only use it to display the large eclipse hero image. Was looking for a "smarter" idea :)

**Comment by paulrobinson** [comment]
@wesbos do you have any ideas for how Markus can improve the code t position the eclipse image (see comments above)?

**Comment by paulrobinson** [comment]
@myfear you might find it easier to maintain the page if the maybe body of text was in an asciidoc file. Let me know if you need me to point you at some examples.

**Comment by myfear** [comment]
@paulrobinson Thanks. I need to look into this the next time. This is ok for now. Getting a final approval from max with the next build. Can we get this live end of this week when I get approval today?

**Comment by paulrobinson** [comment]
OK. I will review it today too.

**Comment by paulrobinson** [comment]
Can you get a better image for SWTBot? It doesn't seem to render too well at that aspect ratio.

**Comment by paulrobinson** [comment]
It might be worth putting the project name under the logos. Currently the reader needs to recognise the logo in order to know which project is being referred to. It's OK ATM for VertX, because the name of the project is clear in the logo. However, all of the others have more obscure logos.

File: eclipse.html.slim [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+	  a.button(href="https://www.eclipse.org/membership/showMember.php?member_id=731") Learn More About The Membership
+
+.row	
+ p
+  .large-5.column.center
+   p &nbsp;
+  .large-19.column
+    h3 Red Hat is a Strategic Developer member of the Eclipse Foundation
+    p 
+     What began as a better way to build software--openness, transparency, collaboration--soon shifted the balance of power in an entire industry. The following table lists the projects we lead. Find out more about all the projects we contribute to, on the
```
Comment (paulrobinson) [inline eclipse.html.slim]: The preceding 'What' is not being rendered on the page.

File: eclipse.html.slim [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+      td
+       a href ="https://eclipse.org/linuxtools/" Linux Tools
+      td 
+       p Alexander Kurtakov
+       a href = "https://twitter.com/akurtakov" 
+        i.fa.fa-lg.fa-twitter-square
+       p and Jeff Johnston
+      td
+       p 
+        The Linux Tools project aims to bring a full-featured C and C++ IDE to Linux developers. We build on the source editing and debugging features of the CDT and integrate popular native development tools such as Valgrind, OProfile, RPM, SystemTap, GCov, GProf, etc. This is the foundation for the
```
Comment (paulrobinson) [inline eclipse.html.slim]: The preceding 'The' is not being rendered.

File: eclipse.html.slim [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+      td
+       p Vert.x is a toolkit for building reactive applications on the JVM
+
+.row	
+ p
+  .large-5.column.center
+   p &nbsp;
+  .large-19.column
+    h3 But There Is Even More To This Partnership
+    p Beside the individual project contributions, Red Hat is also engaged with the Eclipse Foundation and helps on many different levels with governing the platform.
```
Comment (paulrobinson) [inline eclipse.html.slim]: 'Beside the' -> 'Besides the'

Comment (paulrobinson) [inline eclipse.html.slim]: Maybe this would be better:

'in addition to the individual...'

File: eclipse.html.slim [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+    table
+     tr
+      td width = 200
+       p Member in the Eclipse Board Of Directors
+      td width = 200
+       p Max Rydahl-Andersen Red Hat, Inc.
+      td
+       p Max Rydahl Andersen is leading the development of JBoss Tools and Red Hat JBoss Developer Studio and have several years working with Eclipse from both outside and inside of eclipse.org. His main interests for Eclipse.org is to keep the Eclipse IDE a viable platform for building desktop based tooling while also getting involved in offering a complementary cloud and web-based tooling offering.
```
Comment (paulrobinson) [inline eclipse.html.slim]: 'and have several' -> 'and has several'

File: eclipse.html.slim [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+     tr 
+      td 
+       p Member in the Eclipse Project Management Committee
+      td
+       p Aleksandar Kurtakov, Red Hat Inc.
+      td
+       p Aleksandar is part of Red Hat's Eclipse team. He is Platform SWT committer, Tools PMC member and leads the Linux Tools project. He is very interested in deeper native platform integration both as runtime experience and as distribution channel. He is also interested in growing the contributors and better aligning with modern systems.
```
Comment (paulrobinson) [inline eclipse.html.slim]: I'm not sure what you mean by 'deeper native platform integration both as runtime experience and as distribution channel.' It doesn't seem to read right, but I don't understand the statement, so can't suggest an improvement.

**Comment by paulrobinson** [comment]
Otherwise looks fine.

**Comment by myfear** [comment]
@paulrobinson fixed your remarks/bugs and added some more corrections after feedback from Max and Arun. Can you please review again and push live? Thanks.

**Comment by myfear** [comment]
retest this please

**Comment by paulrobinson** [comment]
retest this please

**Pull request merged**